File: src/OK.Hookman.UI/static/src/app/_models/receiver.model.ts
export class ReceiverModel {
    constructor(
        public id: number,
        public name: string,
        public url: string,
        public path: string,
        public headers: {[key: string]: string},
        public queryStrings: {[key: string]: string},
        public createdDate: Date,
        public updatedDate: Date
    ) { }
}

File: src/OK.Hookman.Core/Models/ReceiverModel.cs
using System;
using System.Collections.Generic;

namespace OK.Hookman.Core.Models
{
    public class ReceiverModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Url { get; set; }
        public string Path { get; set; }
        public IDictionary<string, string> Headers { get; set; }
        public IDictionary<string, string> QueryStrings { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; }
    }
}